src/setup/config.rs

```diff
@@ -91,6 +105,21 @@ impl NodeMetaData {
         let mut start_args = args_from(&config_file.start_command);
         let start_command = start_args.remove(0);
 
+        // insert the config file path into start args
+        let config_path = config_file.kind.config_filepath();
+        match config_file.kind {
+            NodeKind::Zebra => {
+                // Zebra's final arg must be `start`, so we insert the actual args before it.
+                let n = start_args.len();
+                assert!(n > 1, "Expected at least one arg for Zebra (`start`)");
```

Comment:
We might actually want to return a result here instead of panicking for future proofing? (A custom runner would likely need `Result`s to function properly.